src/main/java/com/tutorhub/web/controller/ContentController.java

```diff
@@ -1,10 +1,24 @@
-/*
 package com.tutorhub.web.controller;
 
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import com.tutorhub.service.ContentService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
 
 @RestController
 @RequestMapping("/api/v1/content")
-public class ContentController {}
-*/
+@RequiredArgsConstructor
+public class ContentController {
+  private final ContentService contentService;
+
+  @GetMapping(path = "/{fileName}", produces = MediaType.APPLICATION_OCTET_STREAM_VALUE)
+  public Object find(@PathVariable final String fileName) {
+    return contentService.find(fileName);
+  }
+
+  @PostMapping(consumes = MediaType.MULTIPART_FORM_DATA_VALUE)
+  public String upload(@RequestBody MultipartFile file) {
```

Comment:
Did you test this? As I know, Spring Boot accepts multipart via `@RequestParam(name = "file") MultipartFile file`

Reply:
Yes, works well, but I will try your suggestion, will check which is better to use

Reply:
I found that it's recommended to use @RequestParam for MultipartFile

Reply:
Fixed